src/utils/facadeLocator.js

```diff
@@ -0,0 +1,23 @@
+function stripSlashes(path) {
+	return path.replace(/^(\/+)|(\/+)$/g, '');
+}
+
+const setupFacadeLocator = (app) => {
+	app.set('facadeDict', {});
```

Comment:
Mir ist wieder eingefallen warum ich den locator an ein Object bzw. Class binden wollte. Damit man ihn unabhängig von der app in die uc importieren kann.